### PR TITLE
feat: mentorship accept -> send mail

### PR DESCRIPTION
**Pull request by pascaaaal**
### Description
I added a functionality, that sends a mail to the user when the mentorship relation is accepted.

Fixes # [ISSUE]

### Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I just ran the official test <code>python -m unittest discover tests</code> and it worked.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: app/api/dao/mentorship_relation.py [hunk 1]
```diff
@@ -187,6 +191,20 @@ def accept_request(user_id, request_id):
...
+        template = render_template('email_mentorship_accepted.html', user_name=user.name)
+        email_user = UserModel.find_by_id(request.action_user_id)
+
+        msg = Message(
+            "Mentorship System - Mentorship Relation accepted!",
+            recipients=[email_user.email],
+            html=template,
+            sender=application.config['MAIL_DEFAULT_SENDER']
+        )
+        mail.send(msg)
```
Comment (xprilion) [inline app/api/dao/mentorship_relation.py:206]: Hi, can you please use the suitable replacement for this from the `email_utils.py` ?

File: templates/email_mentorship_accepted.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<p>Hi {{ user_name }},</p>
+<br>
+<p>Mentorship Relation accepted!</p>
+<br>
+<p>Congratulations, your Mentorship Relation has been accepted!</p>
```
Comment (xprilion) [inline templates/email_mentorship_accepted.html:5]: Can you please add who accepted the mentorship relationship? Also, having a link to the profile of the mentor would be good for quick actions!

**Review by xprilion — CHANGES_REQUESTED** [review]
Hi, kindly go through my comments on the code :) 

I believe this is near completion but you need to use the provided methods in the existing code for best modularity.

**Comment by sumit-badsara** [comment]
Close due to #327 merge.
Thank you for your contribution!

**Pull request closed without merging**